src/cosmosdb-preview/azext_cosmosdb_preview/_params.py

```diff
@@ -284,3 +284,33 @@ def load_arguments(self, _):
         c.argument('account_name', account_name_type, id_part=None, required=True, help='Name of the CosmosDB database account')
         c.argument('table_name', options_list=['--table-name', '-n'], required=True, help='Name of the CosmosDB Table name')
         c.argument('location', options_list=['--location', '-l'], help="Location of the account", required=True)
+
+    with self.argument_context('cosmosdb dts export') as c:
```

Comment:
```suggestion
    with self.argument_context('cosmosdb dts') as c:
```
You can use `cosmosdb dts` argument context to defined common arguments for all sub argument context, for example, `cosmosdb dts export`, `cosmosdb dts import`. You don't have to repeat `account_name` definition for each command, other arguments either.